workers/unity/Packages/com.improbable.gdk.core/Logging/LogEvent.cs

```diff
@@ -4,13 +4,35 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Represents a single log. Can contain data used for structured logging.
+    /// </summary>
     public struct LogEvent
     {
+        /// <summary>
+        ///     The main content of the log.
+        /// </summary>
         public readonly string Message;
+
+        /// <summary>
+        ///     The data used for structured logging.
```

Comment:
do you mean by structured logging Operate's structured logging or simply having these fields?

Reply:
Yes..?